install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -1,5 +1,30 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
+metadata:
+  name: machine-config-controller
+  namespace: openshift-machine-config-operator
+  labels:
+    k8s-app: machine-config-controller
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  groups:
+    - name: mcc-important-config-paosed
+      rules:
+        - alert: MCCImportantConfigPaused
+          expr: |
+             mcc_important_config_paused > 0
```

Comment:
you will probably want some aggregation here to avoid twitching off and on if/when the machine-config controller container rolls.  Because that will give you a small gap when `mcc_important_config_paused`, and the new container may come up with different injected scrape labels, and both of those may shuffle the time series matching this expression, leading to new alerts (after cooking through [`for`][1]).  I addressed the scrape-labels portion for a CVO-side alert in openshift/cluster-version-operator#643, but missed the metrics-downtime portion.  Something like:

    sum by (description, path, pool) (max_over_time(mcc_important_config_paused[5m])) > 0

should address both issues, but while the `max_over_time(...[5m])` smearing protects you from alert-churning leader blips, it also makes the alert less responsive to the metric actually getting cleared.  So something like "`mcc_important_config_paused` looks bad, or some signal that the machine-config controller is currently not exporting metrics" would be better.

[1]: https://github.com/openshift/machine-config-operator/pull/2802#discussion_r731511094